### PR TITLE
Add new file index.ts

### PR DESCRIPTION
**Pull request by trongle0504**
Add the file to other responsibilities can import to use it

### CODE REVIEW

**Comment by trongle0504** [comment]
Hi @jonahgraham -san,
Could you please help me review it?

Thank you so much!

File: package.json [hunk 1]
```diff
@@ -2,6 +2,8 @@
   "name": "cdt-amalgamator",
   "description": "Amalgamator Debug Adapter from Eclipse CDT",
   "version": "0.0.3",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
```
Comment (jonahgraham) [inline package.json:6]: Should this file be created somewhere? How is it made? If I do `yarn build` it doesn't end up in the `dist` directory.

Comment (trongle0504) [reply package.json:6]: Hi @jonahgraham -san,
I think this file is needed to be able to import "cdt-amalgamator" into other repositories like renesas-gdb-vscode.

Comment (jonahgraham) [reply package.json:6]: Yes, but when I do a `yarn build` it isn't created. Is there something you need to do to create the file? Is the file in your `dist` dir? Do you need to enable `declaration` in tsconfig?

**Comment by jonahgraham** [comment]
I published 0.0.4 with these changes and some fixups I did:

- d9380db Remove the debugAdapter entry point from the exported modules
- 36e23b7 Limit typescript input to src directory

**Pull request merged**